i18n/en.json

```diff
@@ -3097,7 +3097,7 @@
   },
   {
     "id": "api.templates.cloud_trial_ending_email.subtitle",
-    "translation": "{{.Name}}, your 14-day trial of Mattermost Cloud Professional is ending in 3 days, on {{.TrialEnd}}. Please add your payment information to ensure your team can continue enjoying the benefits of Cloud Professional."
+    "translation": "{{.Name}}, your 14-day trial of Mattermost is ending in 3 days. To continue enjoying the benefits of Mattermost Cloud, please add payment information to your account before {{.TrialEnd}}."
```

Comment:
Can we update this text to Justines suggestion here: https://mattermost.atlassian.net/browse/MM-38437?focusedCommentId=116691?

Reply:
There's actually 2 of them. I went with the first

Reply:
updated to the second one

Reply:
Thanks!